templates/index.html

```diff
@@ -14,7 +14,7 @@ <h1>SourceCAD</h1>
   <!--<img class="image-top" src="image/banner.png" alt=""> -->
```

Comment:
Title should be `Computer Aided Dispatch`, not `Computer-aided design`. This goes for all files.